### PR TITLE
Remove the outdated draft document

### PR DESCRIPTION
**Pull request by hayatoito**
As README.md explains, Web Bundles format is now under development in wpack-wg/bundled-responses reposiotry.

Let's remove the outdated draft document in this repository, and update the links to that [1] with the latest draft [2].

- 1: https://wicg.github.io/webpackage/draft-yasskin-wpack-bundled-exchanges.html.
- 2: https://wpack-wg.github.io/bundled-responses/draft-ietf-wpack-bundled-responses.html.

### CODE REVIEW

File: loading.bs [hunk 1]
```diff
@@ -147,7 +147,7 @@ spec: draft-thomson-http-mice; urlPrefix: https://tools.ietf.org/html/draft-thom
     type: dfn
         text: mi-sha256 parameter; url: section-3.1
         text: integrity proof for the first record; url: section-2.2
-spec: draft-yasskin-wpack-bundled-exchanges; urlPrefix: https://wicg.github.io/webpackage/draft-yasskin-wpack-bundled-exchanges.html#
+spec: draft-yasskin-wpack-bundled-exchanges; urlPrefix: https://wpack-wg.github.io/bundled-responses/draft-ietf-wpack-bundled-responses.html#
```
Comment (irori) [inline loading.bs:150]: Can we change `spec: draft-yasskin-wpack-bundled-exchanges` to `spec: draft-ietf-wpack-bundled-responses`?

Comment (hayatoito) [reply loading.bs]: Sure. Done.

**Review by irori — APPROVED** [review]
lgtm

**Comment by hayatoito** [comment]
Thanks for the review. I've rebased the PR on main. Let me merge the PR.

**Pull request merged**